CHANGELOG.md

```diff
@@ -55,6 +55,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 * New command `jj branch move` let you update branches by name pattern or source
   revision.
 
+* `jj git push -c <arg>` can now accept large revsets that resolve to multiple
```

Comment:
nit: drop the "large" since it doesn't seem to add anything and risks making the reader wonder where the boundary between "medium-sized" and "large" is